crates/router/src/connector/fiserv.rs

```diff
@@ -108,6 +238,104 @@ impl
     services::ConnectorIntegration<api::PSync, types::PaymentsSyncData, types::PaymentsResponseData>
     for Fiserv
 {
+    fn get_headers(
+        &self,
+        req: &types::PaymentsSyncRouterData,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+        let timestamp = OffsetDateTime::now_utc().unix_timestamp_nanos() / 1_000_000;
+        let auth: fiserv::FiservAuthType =
+            fiserv::FiservAuthType::try_from(&req.connector_auth_type)?;
+        let api_key = auth.api_key.clone();
+
+        let fiserv_req = self
+            .get_request_body(req)?
+            .ok_or(errors::ConnectorError::RequestEncodingFailed)?;
+        let client_request_id = Uuid::new_v4().to_string();
+        let hmac = self
+            .generate_authorization_signature(auth, &client_request_id, &fiserv_req, timestamp)
+            .change_context(errors::ConnectorError::RequestEncodingFailed)?;
+        let headers = vec![
+            (
+                headers::CONTENT_TYPE.to_string(),
+                types::PaymentsAuthorizeType::get_content_type(self).to_string(),
+            ),
+            ("Client-Request-Id".to_string(), client_request_id),
+            ("Auth-Token-Type".to_string(), "HMAC".to_string()),
+            ("Api-Key".to_string(), api_key),
+            ("Timestamp".to_string(), timestamp.to_string()),
+            ("Authorization".to_string(), hmac),
```

Comment:
please try to use the constants defined in header module, this will help in reducing typos

Reply:
https://github.com/juspay/hyperswitch/blob/8d2e573a29671bbbf74b7646b87c409b25d1b171/crates/router/src/lib.rs#L44